src/test/java/net/snowflake/ingest/streaming/internal/StreamingIngestIT.java

```diff
@@ -117,6 +119,11 @@ public void afterAll() throws Exception {
 
   @Test
   public void testSimpleIngest() throws Exception {
+    // TODO @rcheng - dont want to change factory, so inject
```

Comment:
This test suite uses a factory to create the client. I decided it was better to inject the requestBuilder after it was created by the factory, instead of changing the factory for the purposes of this test. Open to alternatives though!